.travis.yml

```diff
@@ -25,15 +25,29 @@ matrix:
       env: CHECK_CS=1
     - name: "Unit tests"
       env: PHPUNIT_CONFIG='phpunit.xml'
+      php: 7.3
+    - name: "Unit tests (PHP 7.4)"
+      env: PHPUNIT_CONFIG='phpunit.xml'
+      php: 7.4
     - name: "Admin-UI tests on Clean P"
       env:
-      - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
-      - BEHAT_OPTS="--profile=adminui --suite=adminui"
+        - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
```

Comment:
I'd keep this fragment as it used to be, so instead of:
```
    - name: "Admin-UI tests on Clean P"
      env:
        - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
        - BEHAT_OPTS="--profile=adminui --suite=adminui"
        - PHP_IMAGE=ezsystems/php:7.3-v1
      php: 7.3
    - name: "Admin-UI tests on Clean P (PHP 7.4)"
      env:
        - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
        - BEHAT_OPTS="--profile=adminui --suite=adminui"
        - PHP_IMAGE=ezsystems/php:7.4-v1
      php: 7.4
    - name: "Admin UI tests using different personas"
      env:
        - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
        - SETUP_BEHAT_OPTS="--profile=setup --suite=personas"
        - BEHAT_OPTS="--profile=adminui --suite=personas"
        - PHP_IMAGE=ezsystems/php:7.3-v1
      php: 7.3
```
I'd keep:
```
    - name: "Admin-UI tests on Clean P"
      env:
        - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
        - BEHAT_OPTS="--profile=adminui --suite=adminui"
    - name: "Admin UI tests using different personas"
      env:
        - COMPOSE_FILE="doc/docker/base-dev.yml:doc/docker/selenium.yml"
        - SETUP_BEHAT_OPTS="--profile=setup --suite=personas"
        - BEHAT_OPTS="--profile=adminui --suite=personas"
```
There is no need to specify PHP version for Behat jobs as it uses the PHP version inside a container, which is set in .env in metarepo.

Once we update `PHP_IMAGE` and `PHP_IMAGE_DEV` in .env to PHP7.4 I will add separate nightly jobs running on 7.3, so this also will be covered.

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1333/commits/bf603d1fe2118d1352868712b2a71de17e9ccfde